### PR TITLE
Bob/6117 reorg support menu

### LINKED ISSUE #6117: Update Support Admin dashboard categories
## Background
The current **Support Admin** page of SimpleReport ([https://www.simplereport.gov/app/admin](https://www.simplereport.gov/app/admin)) includes several categories for managing organizations, facilities, users, patients, and testing devices. However, the categories are not optimally organized, and some important self-service functionality can be difficult for the Support Staff to find. This issue aims to improve the organization and clarity of the **Support Admin** page so that Support Staff can more easily find and use the necessary functionality.

## Description
The following updates to the categories and page structure will be made to the **Support Admin** page:

### **Organizations & Facilities** 
- **Verify organization identity** (https://www.simplereport.gov/app/admin/pending-organizations)
- **Add organization admin** (https://www.simplereport.gov/app/admin/add-organization-admin)
- **Delete/Undelete a facility** _(need link)_
- **Ghost into an organization** (https://www.simplereport.gov/app/admin/tenant-data-access)

### **Users & Patients** 
- **Manage users** _(need link)_
- **Unarchive patient** _(need link)_

### **Devices** 
- **Add a new device** (https://www.simplereport.gov/app/admin/create-device-type)
- **Edit an existing device** (https://www.simplereport.gov/app/admin/manage-devices)

## Acceptance Criteria

- The **Support Admin** page ([https://www.simplereport.gov/app/admin](https://www.simplereport.gov/app/admin)) has been updated to reflect the new organization and ordering of categories.
- Page has also been updated with new wording of existing links.
- The Support Staff have been notified of the updates made to the **Support Admin** page.
- In the case that the new functionality is not yet available, the link will not be displayed on the page until after the new feature has been shared with the Support Team as a demo.

## Open questions to address before merging

- [ ] Final signoff from Laura Kerry on copy
- [ ] Also signoff from Laura Kerry re: link text matching page title text - should the links always match? if so, let's clarify what each of these pages should be titled

### PR DESCRIPTION
**Pull request by fzhao99**
# FRONTEND PULL REQUEST

## Related Issue
Fixes #6117 

## Changes Proposed

- Copy / home page reorg changes for the support admin pages
    - I got annoyed that I was changing strings across a bunch of different files so just pulled everything into a constant file and exported the vars. 
    - Cypress seems to have trouble compiling ts files when I tried to import the page title constants, so in the e2e tests we're just using the raw strings
    - Also changed the page titles for the admin pages to match the links on suggestion
- Update page titles in Azure Insights hooks / tests to match
- Will leave this unmerged until we get the final OK from support.

Before 
Screenshot 2023-07-20 at 12 09 24 PM

After
Screenshot 2023-07-24 at 10 55 36 AM

## Testing

- Verify copy changes match suggestions [in ticket](https://github.com/CDCgov/prime-simplereport/issues/6117#issuecomment-1644291485)

### Content
- [x] Any content changes have been approved by content team

### Support
- [x] Any changes that might generate new support requests have been flagged to the support team
- [x] Any changes to support infrastructure have been demo'd to support team

### CODE REVIEW

**Review by johanna-skylight — APPROVED** [review]
LGTM!

File: frontend/src/app/supportAdmin/pageTitles.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// js file so imports to cypress work
+export const addNewDevicePageTitle = "Add a new device";
```
Comment (emyl3) [inline frontend/src/app/supportAdmin/pageTitles.ts:2]: looks like based on Laura's comments, this was suppose to be "Add new device" without the "a"

File: frontend/src/app/supportAdmin/pageTitles.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+export const editDevicePageTitle = "Edit existing device";
+export const identityVerificationPageTitle =
+  "Verify/edit organization identity";
+export const orgAccessPageTitle = "Access organization account";
+export const addOrgAdminPageTitle = "Add organization admin";
+
+export const orgFacilityColumnTitle = "Organizations & facilities";
+export const devicesColumnTitle = "Devices";
+export const usersAndPatientsColumnTitle = "Users & patients";
```
Comment (emyl3) [inline frontend/src/app/supportAdmin/pageTitles.ts:11]: for "patients" we can import that as `PATIENT_TERM_PLURAL` from the `src/config/constants.ts` file

**Review by emyl3 — CHANGES_REQUESTED** [review]
Left a couple comments! Thank you, Bob!

File: frontend/src/app/supportAdmin/pageTitles.ts (hunk 1 above)
Comment (fzhao99) [reply frontend/src/app/supportAdmin/pageTitles.ts]: thanks for the catch elisa!

File: frontend/src/app/supportAdmin/DeviceType/__snapshots__/DeviceTypeFormContainer.test.tsx.snap [hunk 3]
```diff
@@ -23,7 +23,7 @@ Object {
                 <h1
                   class="font-heading-lg margin-top-0 margin-bottom-0"
                 >
-                  Device type
+                  Add a new device
```
Comment (emyl3) [inline frontend/src/app/supportAdmin/DeviceType/__snapshots__/DeviceTypeFormContainer.test.tsx.snap:26]: we need to update this snapshot as well

File: frontend/src/app/supportAdmin/__snapshots__/SupportAdmin.test.tsx.snap [hunk 4]
```diff
@@ -84,19 +84,33 @@ exports[`SupportAdming loads menu categories 1`] = `
                       class=""
                       href="/admin/create-device-type"
                     >
-                      Add a new testing device
+                      Add a new device
```
Comment (emyl3) [inline frontend/src/app/supportAdmin/__snapshots__/SupportAdmin.test.tsx.snap:87]: same here

**Review by emyl3 — CHANGES_REQUESTED** [review]
left a couple more comments to get the tests and e2e passing :D Thank you, bob!

**Review by emyl3 — DISMISSED** [review]
LGTM!

File: frontend/src/app/Settings/Facility/Components/ManageDevices.tsx [hunk 5]
```diff
@@ -31,7 +32,7 @@ const ManageDevices: React.FC<Props> = ({
   return (
     <div className="prime-container card-container">
       <div className="usa-card__header">
-        <h2 className="font-heading-lg">Manage devices</h2>
+        <h2 className="font-heading-lg">{editDevicePageTitle}</h2>
```
Comment (mehansen) [inline frontend/src/app/Settings/Facility/Components/ManageDevices.tsx:35]: this is the title for selecting devices for a facility in the manage facility page - I don't think swapping this for the new copy makes sense here

File: frontend/src/app/supportAdmin/pageTitles.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+export const addNewDevicePageTitle = "Add new device";
+export const editDevicePageTitle = "Edit existing device";
+export const identityVerificationPageTitle =
+  "Verify/edit organization identity";
+export const orgAccessPageTitle = "Access organization account";
+export const addOrgAdminPageTitle = "Add organization admin";
+
+export const orgFacilityColumnTitle = "Organizations & facilities";
+export const devicesColumnTitle = "Devices";
+export const usersAndPatientsColumnTitle = `Users and ${PATIENT_TERM_PLURAL}`;
```
Comment (mehansen) [inline frontend/src/app/supportAdmin/pageTitles.ts:13]:
```suggestion
export const usersAndPatientsColumnTitle = `Users & ${PATIENT_TERM_PLURAL}`;
```

**Review by mehansen — COMMENTED** [review]
couple small fixes

can we make sure either you or Jayna has updated support on the copy before we merge? I think everything matches what they already saw, but the link name for ghosting has changed again so would be good for them to know

**Review by mehansen — APPROVED** [review]
awesome! thanks for the changes

**Review by johanna-skylight — APPROVED** [review]

**Pull request merged**